contrib/roles/windows/docker/tasks/requirements-16299.yaml

```diff
@@ -8,9 +8,9 @@
   register: features_installed
   with_items:
     - Containers
-    - Hyper-V # TODO
-    - RSAT-Hyper-V-Tools # TODO
-    - Hyper-V-PowerShell # TODO
+    #- Hyper-V # TODO
+    #- RSAT-Hyper-V-Tools # TODO
+    #- Hyper-V-PowerShell # TODO
```

Comment:
Just remove those lines instead of commenting them.

Reply:
Hmmm. Should we bother we Hyper-V features via some variable?